Documentation/cmdref/clustermesh-apiserver_kvstoremesh.md

```diff
@@ -16,10 +16,11 @@ clustermesh-apiserver kvstoremesh [flags]
       --cluster-name string                          Name of the cluster. It must consist of at most 32 lower case alphanumeric characters and '-', start and end with an alphanumeric character. (default "default")
       --clustermesh-config string                    Path to the ClusterMesh configuration directory
       --controller-group-metrics strings             List of controller group names for which to to enable metrics. Accepts 'all' and 'none'. The set of controller group names available is not guaranteed to be stable between Cilium versions.
+      --crd-wait-timeout duration                    Cilium will exit if CRDs are not available within this duration upon startup (default 5m0s)
```

Comment:
This flag is not applicable to KVStoreMesh, and should not be registered here.

Reply:
Sorry, yeah I should have caught this during my own self-review. Thanks for the detailed review, I'll address your feedback.